### PR TITLE
text utilities

### PR DESCRIPTION
**Pull request by rajaranjanmr**

### CODE REVIEW

File: buttons/button.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+.btns button{
+    outline: 0;
+    padding-left: 2rem;
+    padding-right: 2rem;
+    padding-top: 0.7em;
+    padding-bottom: 0.5rem;
+    color: var(--button-content-color);
+    background-color: var(--button-background-color);
+    text-align: center;
+    cursor: pointer;
```
Comment (manish-GH) [inline buttons/button.css]: It would be better to add cursor: pointer on button hover

File: buttons/button.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+.btns a{
+    padding-left: 2rem;
+    padding-right: 2rem;
+    padding-top: 0.7em;
+    padding-bottom: 0.5rem;
+    color: var(--button-content-color);
+    background-color: var(--button-background-color);
+    text-align: center;
+    cursor: pointer;
```
Comment (manish-GH) [inline buttons/button.css]: same here, add this on hover

File: cards/card1.css [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+.para .subheading{
+    padding-top:0.2rem;
+    padding-bottom:0.2rem;
+    font-weight: 200;
+    margin-left:0.3rem;
+}
+.para .price{
+    position: absolute;
+    right:8px;
+    bottom:45px;
```
Comment (manish-GH) [inline cards/card1.css]: use rem

File: cards/card1.css [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+}
+/*  */
+.cards{
+    margin:1rem;
+}
+.badge{
+    width:35%;
+    height: auto;
+    background-color: rgb(191, 194, 5);
```
Comment (manish-GH) [inline cards/card1.css]: use variables for colors

File: cards/card1.css [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+    text-align: center;
+    border-radius: 1rem;
+    padding-right: 1rem;
+    padding-left:1rem;
+    margin-bottom:0.5rem;
+    padding-bottom:0.3rem;
+}
+.card-btn:hover{
+    transition: 5s;
+    background-color: rgb(57, 8, 104);
```
Comment (manish-GH) [inline cards/card1.css]: use variable

File: cards/cardWithBadge.css [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+.para-card .subheading-card{
+padding-top:0.2rem;
+padding-bottom:0.2rem;
+font-weight: 200;
+margin-left:0.3rem;
+}
+.para-card .price-card{
+position: absolute;
+right:8px;
+bottom:45px;
```
Comment (manish-GH) [inline cards/cardWithBadge.css]: use rem

File: cards/cardWithBadge.css [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+background-color: var(--card-badge-rating-bg);
+display: inline;
+padding:0.3rem;
+padding-left:0.5rem;
+padding-right:0.5rem;
+border-radius:5px;
+color:var(--card-background-color);
+position: absolute;
+right:8px;
+bottom:70px;
```
Comment (manish-GH) [inline cards/cardWithBadge.css]: use rem

File: cards/cardwithDismiss.css [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+}
+.para-card p{
+text-align: justify;
+padding: 0.3rem;
+margin-top:0.3rem;
+margin-bottom:0.3rem;
+}
+.card-btn{
+padding-top:0.2rem;
+background-color: #082032;
```
Comment (manish-GH) [inline cards/cardwithDismiss.css]: use variable

File: cards/cardwithShadow.css [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+.para-card .subheading-card{
+    padding-top:0.2rem;
+    padding-bottom:0.2rem;
+    font-weight: 200;
+    margin-left:0.3rem;
+}
+.para-card .price-card{
+    position: absolute;
+    right:8px;
+    bottom:45px;
```
Comment (manish-GH) [inline cards/cardwithShadow.css]: use rem

File: cards/cardwithhorizontal.css [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+    padding:0;
+    margin:0;
+    box-sizing: border-box;
+    font-family: sans-serif;
+    letter-spacing: 1px;
+}
+.card-legend-hori{
+    width:100%;
+    border:1px solid black;
+    background-color: aliceblue;
```
Comment (manish-GH) [inline cards/cardwithhorizontal.css]: use variable

File: cards/cardwithoverlay.css [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+.para-card .subheading-card{
+padding-top:0.2rem;
+padding-bottom:0.2rem;
+font-weight: 200;
+margin-left:0.3rem;
+}
+.para-card .price-card{
+position: absolute;
+right:8px;
+bottom:45px;
```
Comment (manish-GH) [inline cards/cardwithoverlay.css]: use rem

File: cards/cardwithtext.css [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+letter-spacing: 1px;
+}
+.card-legend{
+width:20%;
+border:1px solid var(--card-border-color);
+background-color: var(--card-background-color);
+position: relative;
+border-radius:10px;
+overflow:hidden;
+min-height:255px;
```
Comment (manish-GH) [inline cards/cardwithtext.css]: use rem

File: inputBox/index.css [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+    align-content: center;
+    border-radius:1rem;
+    padding-right: 2rem;
+    
+}
+input[type="text"]{
+    width: 100%;
+    padding:0.7rem;
+    font-weight: lighter;
+    font-size:20px;
```
Comment (manish-GH) [inline inputBox/index.css:29]: use rem

File: inputBox/index.css [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+    border-radius:1rem;
+    text-align: top;
+
+
+}
+input[type="text"]:hover{
+    border-color:tomato;
+}
+label{
+    font-size:20px;
```
Comment (manish-GH) [inline inputBox/index.css]: use rem

File: inputBox/index.css [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+    font-weight: lighter;
+    text-align: center;
+    position: relative;
+    right: -23px;
```
Comment (manish-GH) [inline inputBox/index.css]: use rem

File: inputBox/index.css [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+    display:inline;
+}
+input[type="password"]{
+    width: 100%;
+    padding:0.7rem;
+    font-weight: lighter;
+    font-size:20px;
```
Comment (manish-GH) [inline inputBox/index.css]: use rem

File: textstyles/texts.html [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="texts.css">
+</head>
+<body>
+    <div class="text-utilities">
+        <div class="h1">
```
Comment (manish-GH) [inline textstyles/texts.html:12]: I don't think you need both div and h tags. You can use either 

`<div class="h1">Heading 1</div>`
or reset the h1 tag according to your need and just use
`<h1>Heading 1</h1>`

Same for all the other H tags

File: textstyles/texts.html [hunk 18]
```diff
@@ -0,0 +1,42 @@
...
+        <div class="h3">
+            <h3>This is heading 3</h3>
+        </div>
+        <div class="h4">
+            <h4>This is heading 4</h4>
+        </div>
+        <div class="h5">
+            <h5>This is heading 5</h5>
+        </div>
+        <div class="h5">
```
Comment (manish-GH) [inline textstyles/texts.html]: the class should be h6

File: buttons/button.html [hunk 19]
```diff
@@ -0,0 +1,32 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="button.css">
+    <link rel="stylesheet" href="">
+</head>
+<body>
+    <div class="buttons">
+        <div class="primary-btn btns">
```
Comment (manish-GH) [inline buttons/button.html:14]: For the class names, we were suggested to his convention
btn btn-primary

Follow the same for other buttons as well

**Review by manish-GH — CHANGES_REQUESTED** [review]
Also please add the navigation function for the sidebar.

File: inputBox/index.css [hunk 20]
```diff
@@ -0,0 +1,64 @@
+*{
+    padding:0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: sans-serif;
+    background-color: #041562;
```
Comment (manish-GH) [inline inputBox/index.css]: use variables for colors

File: cards/card1.css [hunk 21]
```diff
@@ -0,0 +1,205 @@
...
+    margin-left:0.3rem;
+}
+.para .price{
+    position: absolute;
+    right:8px;
+    bottom:2.8rem;
+    font-weight: 200;
+}
+.para .rating .rating-h{
+    background-color: green;
```
Comment (manish-GH) [inline cards/card1.css]: use variable

File: cards/card1.css [hunk 22]
```diff
@@ -0,0 +1,205 @@
...
+    text-align: center;
+    border-radius: 1rem;
+    padding-right: 1rem;
+    padding-left:1rem;
+    margin-bottom:0.5rem;
+    padding-bottom:0.3rem;
+}
+.card-btn:hover{
+    transition: 5s;
+    background-color:#390868;
```
Comment (manish-GH) [inline cards/card1.css]: use variable

File: cards/cardwithShadow.css [hunk 23]
```diff
@@ -0,0 +1,105 @@
...
+    background-color: var(--card-badge-rating-bg);
+    display: inline;
+    padding:0.3rem;
+    padding-left:0.5rem;
+    padding-right:0.5rem;
+    border-radius:5px;
+    color:var(--card-background-color);
+    position: absolute;
+    right:8px;
+    bottom:70px;
```
Comment (manish-GH) [inline cards/cardwithShadow.css]: use rem

File: cards/cardwithoverlay.css [hunk 24]
```diff
@@ -0,0 +1,108 @@
...
+}
+.para-card p{
+text-align: justify;
+padding: 0.3rem;
+margin-top:0.3rem;
+margin-bottom:0.3rem;
+}
+.card-btn{
+padding-top:0.2rem;
+background-color: #082032;
```
Comment (manish-GH) [inline cards/cardwithoverlay.css]: use variable

File: inputBox/index.css [hunk 25]
```diff
@@ -0,0 +1,64 @@
...
+}
+.formInput{
+    width:45%;
+    background-color: #f2f2f2;
```
Comment (manish-GH) [inline inputBox/index.css]: use variable

File: inputBox/index.css [hunk 26]
```diff
@@ -0,0 +1,64 @@
...
+    font-weight: bold;
+    background-color: #4CAF50;
+    border-radius: 2rem;
+    margin:1rem;
+    color:white;
+    font-size: 15px;
+
+}
+.submitBtn:hover{
+    background-color: #45a049;
```
Comment (manish-GH) [inline inputBox/index.css]: use variable

File: inputBox/index.css [hunk 27]
```diff
@@ -0,0 +1,64 @@
...
+    font-weight: lighter;
+    font-size:20px;
+    margin: 1rem;
+    border-radius:1rem;
+    text-align: top;
+
+
+}
+input[type="text"]:hover{
+    border-color:tomato;
```
Comment (manish-GH) [inline inputBox/index.css]: use variable

File: inputBox/index.css [hunk 28]
```diff
@@ -0,0 +1,64 @@
...
+    width: 100%;
+    padding:0.7rem;
+    font-weight: lighter;
+    font-size:1.62rem;
+    margin: 1rem;
+    border-radius:1rem;
+    text-align: top;
+}
+input[type="password"]:hover{
+    border-color:tomato;
```
Comment (manish-GH) [inline inputBox/index.css]: use variable

**Review by manish-GH — CHANGES_REQUESTED** [review]

**Review by manish-GH — APPROVED** [review]
You have missed some, have a look at line-29 in inputBox/index.css file and line-12 in textstyles/text.html file.

Everything else seems fine, can be merged

File: badges/badgesAvtaar.css [hunk 29]
```diff
@@ -42,43 +46,38 @@
...
     height: 25%;
     position: relative;
 
 }
-.lg{
+.lg-badges{
     width: 30%;
     height: 30%;
     position: relative;
     border-radius: 50%;
```
Comment (sri-rishi) [inline badges/badgesAvtaar.css]: remove  extra spaces

File: cards/card1.html [hunk 30]
```diff
@@ -0,0 +1,127 @@
...
+        </div>
+        <div class="para">
+            <p class="paragraph">Lorem ipsum dolor sit amet consectetur adipisicing elit. Sed blanditiis, 
+                consectetur officiis perspiciatis nostrum explicabo esse totam eius facere maiores repellat modi 
+                quibusdam cum velit aperiam in veritatis iusto consequuntur?</p>
+        </div>
+        <div class="card-button">
+            <button class="card-btn">show</button>
+            </div>
+    </div>
```
Comment (sri-rishi) [inline cards/card1.html]: if not necessary remove comments

File: cards/cardWithBadge.css [hunk 31]
```diff
@@ -0,0 +1,103 @@
...
+height: auto;
+background-color: var(--card-badge-bg-col);
+padding:0.7rem;
+padding-left:1rem;
+padding-right: 1rem;
+text-align: center;
+border-radius:5px;
+position: absolute;
+top: 1rem;
+
```
Comment (sri-rishi) [inline cards/cardWithBadge.css]: give tab spacing here hard to differentiate between properties and class name

File: responsive images/responsiveimage1.css [hunk 32]
```diff
@@ -0,0 +1,15 @@
+*{
+    padding:0;
+    margin:0;
+}
+.img-container{
+  width:100%;
+  display:flex;
+  flex-direction:row;
+  gap:1rem;
```
Comment (sri-rishi) [inline responsive images/responsiveimage1.css:9]: Be consistent with spacing

File: responsive images/round.html [hunk 33]
```diff
@@ -0,0 +1,36 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        .resp-round-img{
+    width: 100%;
+    height: auto;
+    text-align: center;
```
Comment (sri-rishi) [inline responsive images/round.html]: keep styles in separate  file

**Review by sri-rishi — CHANGES_REQUESTED** [review]

**Review by sri-rishi — APPROVED** [review]

**Pull request merged**